<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>corner_coors_dialog</class>
 <widget class="QDialog" name="corner_coors_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>244</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enter Corner Coordinates</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>Please enter the corner coordinates of you chart</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="title_seperator_line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="input_grid">
     <item row="0" column="2">
      <widget class="QLabel" name="lon_label">
       <property name="text">
        <string>Longitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="top_left_label">
       <property name="text">
        <string>Top Left</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="btm_right_lat_input"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="btm_right_lon_input"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lat_label">
       <property name="text">
        <string>Latitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="top_left_lat_input"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="top_left_lon_input"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="btm_right_label">
       <property name="text">
        <string>Bottom Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="cancelok_spcr">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
